Core/DISET/ServiceReactor.py

```diff
@@ -151,6 +151,13 @@ def __getListeningSocketsList( self, svcName = False ):
     return sockets
 
   def __acceptIncomingConnection( self, svcName = False ):
+    """
+      This method get incomming connections.
+      This method just get the incomming connection, check IP address
+      and generate job. SSL/TLS handshake and execution of remote call
+      are made by Service._processInThread() (in another thread) so
+      DIRAC can accept other client while Service handle remote call
```

Comment:
this method getS, checkS, generateS etc

Tu peux aussi ajouter le dockstring ici ?